### PR TITLE
feat: hexless account profiles register and update RPC specs

### PR DESCRIPTION
**Pull request by geekbrother**
Registering hexless account name and updating it.
Adding and updating addresses for the name.

### CODE REVIEW

**Review by chris13524 — APPROVED** [review]
LGTM

File: docs/specs/servers/blockchain/blockchain-server-api.md [hunk 1]
```diff
@@ -106,3 +108,45 @@ Used to lookup for suggestions for the new name by returning 3 unique usernames.
...
+```
+
+* `message` - JSON serialized string that should contain the following fields:
+
+    * `name` - is the name to register or update.
+        * Alphabetic characters (A-Z a-z), numeric characters (0-9), the minus sign (-) and the period (.)
+        * Max 255 characters length.
+    * `address` - address that bind to the account name.
+        * Max 100 characters length.
+    * `timestamp` - current unixtime timestamp. The signature is valid for 30 seconds.
```
Comment (geekbrother) [inline docs/specs/servers/blockchain/blockchain-server-api.md]: Adding the 30 seconds validity of the signature, to limit replay attack.

Comment (chris13524) [reply docs/specs/servers/blockchain/blockchain-server-api.md]: Might be a bit short to accommodate clock drift, I'd do 5 minutes to be safe.

Comment (geekbrother) [reply docs/specs/servers/blockchain/blockchain-server-api.md]: The reference for the 30 seconds for my decision is the [TOTP](https://ru.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm) it's time-based and it has a 30 seconds gap for the sync.

**Pull request merged**